main.go

```diff
@@ -30,18 +31,22 @@ func Init() {
 }
 
 func main() {
-	Init()
 	r := router.New()
 	r.GET("/", Index)
 	r.GET("/hello/{name}", Hello)
 	r.POST("/insertSingle", controllers.SendSingleToChannel)
 	r.POST("/insertMultiple", controllers.SendMultipleToChannel)
 	r.POST("/shutdownServiceAsap", controllers.ShutdownService)
 
-	err := resources.InitDatabase()
+	err := resources.InitDB()
 	if err != nil {
 		log.Fatal("E#1KDZRP - " + err.Error())
 	}
 	go dbLogWriter.StartWritingLogs()
-	log.Fatal(fasthttp.ListenAndServe(":8080", r.Handler))
+	port := os.Getenv("APP_PORT")
+	if len(port) == 0 {
+		port = ":8080"
```

Comment:
People don't set port numbers as `:8080`. They set it as `8080`

Reply:
Made the change!